mchf-eclipse/src/uhsdr_main.c

```diff
@@ -46,6 +46,7 @@
 #include "drivers/ui/oscillator/osc_si570.h"
 #include "drivers/audio/codec/codec.h"
 #include "misc/profiling.h"
+#include "uhsdr_canary.h"
```

Comment:
You could remove including malloc.h up there also, I think...

Reply:
Done.